apps/wallet-connect/package.json

```diff
@@ -5,6 +5,7 @@
   "homepage": "./",
   "dependencies": {
     "@gnosis.pm/safe-react-components": "^0.9.7",
+   "@gnosis.pm/safe-react-gateway-sdk": "^2.8.5",
```

Comment:
Did we sync with the infra team if it's ok to use the gateway? so far, the gateway was meant to be used in clients only

Reply:
We synced, is expected that in the future will be a gate for external devs to fetch info directly from backend. Only limitation they warned us is that client-gateway is meant for Safe clients and thereby backward compatibility is not guaranteed, endpoints may break in future versions. 

As we are integrating with the same package needed for web we would be fine in terms of adapting to gateway changes and both teams can help each other to keep it updated